sqlmodel/main.py

```diff
@@ -531,13 +529,9 @@ def from_orm(cls: Type["SQLModel"], obj: Any, update: Dict[str, Any] = None):
         if update is not None:
             obj = {**obj, **update}
         # End SQLModel support dict
-        if not getattr(cls.__config__, "table", False):
-            # If not table, normal Pydantic code
-            m = cls.__new__(cls)
-        else:
-            # If table, create the new instance normally to make SQLAlchemy create
-            # the _sa_instance_state attribute
-            m = cls()
+        m = cls.__new__(cls) if not getattr(cls.__config__, "table", False) else cls()
+        # If table, create the new instance normally to make SQLAlchemy create
+        # the _sa_instance_state attribute
```

Comment:
In this case I prefer the old version, where the comment is right on top of the thing that it's doing.